content/guides/getstarted/how-to-create-a-token.md

```diff
@@ -0,0 +1,329 @@
+
+```json
+{
+  "name": "",
+  "symbol": "",
+  "image": ""
+}
+```
+
+The meta data we will save in a decentralized storage solution.
+
+First we need to create an icon for the token. I used ChatGPT to generate an
+image and then uploaded it to a decentralized storage solution.
+
+There are many ways on how you can store the offchain meta data. You can for
+examples just upload it to github, or any other cloud storage provider. It just
+needs to be publicly accessible.
+
+Following I will show you different options in no particular order.
+
+| Service              | URL                                                      | Details                                                                                             |
+| -------------------- | -------------------------------------------------------- | --------------------------------------------------------------------------------------------------- |
+| Web3 Storage         | [web3.storage](https://web3.storage)                     | Requires signing up for a free plan; first 5Gb are free; easy to use.                               |
+| Akord                | [Akord](https://akord.com/)                              | Uploads to arweave; free without signup for 100Mb; uploads can take a while.                        |
+| GitHub               | [GitHub](https://github.com)                             | Create a new repo, upload files, use the RAW URL format: https://raw.githubusercontent.com/xxxxxxx. |
+| ShadowDrive          | [ShadowDrive](https://www.shdwdrive.com/)                | This is a solana native storage solution. Did not work at the time of writing.                      |
+| Metaplex Metaboss    | [Metaplex Metaboss](https://metaplex.com/)               | [Meta boss docs](https://metaboss.rs/)-                                                             |
+| Google Cloud Storage | [Google Cloud Storage](https://cloud.google.com/storage) | -                                                                                                   |
+| Amazon               | [Amazon](https://aws.amazon.com/)                        | -                                                                                                   |
+
+(If you are a provider of decentralized storage solutions and want to be listed
+here please just open a PR using the edit page button)
+
+For this example we will use Web3 Storage. It requires to sign up for a free
+plan though but the first 5Gb are free and its very easy to use.
+
+- [web3.storage](web3.storage)
+
+Once we have generated and uploaded the icon to a decentralized storage solution
+we end up with a link to that icon. We will use that link in the meta data file.
+
+You will end up with some link similar to this one which should directly open
+your icon: https://arweave.net/itK2SKyCDAdBl-t9sHDQzeP4Roh3UgMaqnKImRXSrvo
+
+Create a new file called `metadata.json` in the `nice-token` folder and add the
+following:
+
+```json
```

Comment:
```suggestion
```json filename="metadata.json"
```

Reply:
Ah sweet I didn't know we can do this, I will steal it for docs.